content/id/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -0,0 +1,414 @@
+---
+title: Konfigurasi Quality of Service untuk Pod
+content_type: task
+weight: 30
+---
+
+<!--
+title: Configure Quality of Service for Pods
+content_type: task
+weight: 30
+-->
+
+<!-- overview -->
+
+<!--
+This page shows how to configure Pods so that they will be assigned particular
+Quality of Service (QoS) classes. Kubernetes uses QoS classes to make decisions about
+scheduling and evicting Pods.
+-->
```

Comment:
This is a great way to translate and ease reviewing (leave the commented english sentences)! 
however, let's clean them up once we are satisfied with the translation.